server/routes/ActivityType.route.js

```diff
@@ -0,0 +1,25 @@
+/////////////////////// Import dependencies /////////////////////
+const express = require('express');
+const activityTypeController = require('../controllers/ActivityType.controller');
+const router = express.Router();
+/////////////////////////////////////////////////////////////////
+
+
+/////////////////////////// Routes ///////////////////////////
+router.route('/all')
+.get(activityTypeController.getAllActivityTypes);
```

Comment:
Will need API_KEY verification called before crud operations can run for all routes

Reply:
Done with JWT instead.